### ISSUE TITLE
Learning rate and ngpus

### ISSUE THREAD

**Issue by cientgu** [rosinality/stylegan2-pytorch#225]
The default learning rate is 0.002 when I use 4 GPUs, each of them adopts the batch size number as 8. When I use 8 GPUs, and each of them still has 8 samples in a batch, should I enlarge the learning rate? Such as changing to 0.004?

**Comment by rosinality** [comment]
I am not sure about linear lr rule works well in gan trainings. I think you can just use 0.002 or use 4 sample per gpus.

**Comment by cientgu** [comment]
> I am not sure about linear lr rule works well in gan trainings. I think you can just use 0.002 or use 4 sample per gpus.

Thanks for your suggestion!
